### ISSUE TITLE
Infinite packet length mode

### ISSUE THREAD

**Issue by LennP** [fphammerle/python-cc1101#141]
Hi,

I am trying to use this library to send packets that are about 600 bytes long using a CC1101. I see the PacketLengthMode.INFINITE has been commented out in the options.py file. Is this because it has not yet been implemented?

Best regards,
Lenn

**Comment by fphammerle** [comment]
Hi,

yes, currently the library does not yet support `PacketLengthMode.INFINITE`  (missing implementation).

Use of `PacketLengthMode.INFINITE` is documented in section 15.2.2 "Packet Length > 255" of CC1101's manual: https://www.ti.com/lit/ds/symlink/cc1101.pdf

With the current implementation you could use `CC1101.asynchronous_transmission()` instead.
However, that mode requires manual encoding and timing of the signal (by setting pin `GDO0`)

**Comment by LennP** [comment]
Hi,

Thanks for your reply. So if I'm right you have not given it a try to implement it, therefore it would likely be possible?

**Comment by fphammerle** [comment]
Yes, I haven't tried yet but I assume that it is possible with a few hours of effort
